usr/src/uts/common/fs/zfs/vdev.c

```diff
@@ -22,9 +22,9 @@
 /*
  * Copyright (c) 2005, 2010, Oracle and/or its affiliates. All rights reserved.
  * Copyright (c) 2011, 2015 by Delphix. All rights reserved.
- * Copyright 2015 Nexenta Systems, Inc.  All rights reserved.
  * Copyright (c) 2014 Integros [integros.com]
  * Copyright 2016 Toomas Soome <tsoome@me.com>
+ * Copyright 2017 Nexenta Systems, Inc.
```

Comment:
While it isn't wrong to do it this way, it would be easier for others to merge their adjacent copyright changes if you kept the Nexenta copyright on the same line, and just updated the date/message.